### DISCUSSION TITLE
How to implement Clover with Arch Linux?

### DISCUSSION THREAD

**Discussion by sumduck** [CloverHackyColor/CloverBootloader#489] [Q&A]
How do I get Arch Linux to work with the Clover Bootloader? I'm honestly very confused. I read the [ArchWiki ](https://wiki.archlinux.org/title/Clover)article on it, as everyone I've asked has told me to, but it just tells you to do things, now how to do them. And when I look up how to do those things, I can never figure out how exactly to do it. 

If someone could point me to an article or link that has a step-by-step tutorial on how to do this implementation, that would be 
greatly appreciated.

**Comment by Morbius01** [comment]
I have successfully installed Linux Mint and Kali Linux to boot with Clover. So, I guess, for Arch Linux, it should be a similar process.
See my post [here](https://github.com/CloverHackyColor/CloverBootloader/discussions/491)

So, I might be able to help, if you are still interested.
Have you managed to install Arch Linux to boot with Clover ?
